amplience-automation/automation-files/content/sitestructure/01 - Main Navigation/Our Jewelry.json

```diff
@@ -0,0 +1,40 @@
+{
+  "id": "73249be6-4dd4-43dc-b4f9-d3ffffe7340c",
+  "body": {
+    "_meta": {
+      "name": "Jewelry 2",
```

Comment:
Maybe this should be changed to "Our Jewelry"?

Reply:
Yes